views/default/output/url.php

```diff
@@ -57,9 +57,15 @@
 	}
 
 	if (!elgg_extract('is_trusted', $vars, false)) {
```

Comment:
Hmm, logic isn't quite right. If `$vars['is_trusted']` was set to `false` we want to take it on its word, right?